python_exercise/db.py

```diff
@@ -127,3 +129,37 @@ def get_item(item_type: ItemType, tenant_id: str, item_id: str, fields=None) ->
             raise ItemNotFound(item_type.value, tenant_id, item_id)
 
         return response.get("Item").get("data")
+
+    @staticmethod
+    @start_span("database_update_item")
+    def update_item(item_type: ItemType, tenant_id: str, item_id: str, item_data: Mapping[str, Any] = None):
+        """
+        Store updated item information in database
+        :param item_type: One of the types from ItemType
+        :param tenant_id: item tenant
+        :param item_id: item id
+        :param item_data: data to store with item
+        """
+        logger.info(f"Updating item from DB for item [{item_id}] for tenant [{tenant_id}]")
```

Comment:
I think a message like this would look a bit cleaner:
```suggestion
        logger.info(f"Updating item [{item_id}] for tenant [{tenant_id}]")
```
(Also note: just because someone comments a suggestion on your PR, that doesn't mean you have to listen to it - if you disagree with suggested changes, it's common to either reply to the post and explain why you disagree, or directly talk to the person about the suggestion and reach an understanding on what to do)

Reply:
Noted the suggestion i also think your change looks cleaner so i removed the extra words